.travis.yml

```diff
@@ -26,3 +26,6 @@ cache:
   yarn: true
   directories:
   - node_modules
+
+notifications:
+  email: false
```

Comment:
Fine with me. It might be nice to know if master ever breaks. There might be config to keep just one branch.